media-gfx/yacreader/patches/yacreader-9.5.0.patchset

```diff
@@ -0,0 +1,147 @@
+From e277b4771fc6b3e9136a434742e3b8ed9fbd07a0 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Thu, 27 Sep 2018 10:48:01 +0200
+Subject: [PATCH] Install adjustments for HAIKU
+
+---
+ YACReader/YACReader.pro               | 24 ++++++++++++++++++++----
+ YACReaderLibrary/YACReaderLibrary.pro | 24 ++++++++++++++++++++----
+ 2 files changed, 40 insertions(+), 8 deletions(-)
+
+diff --git a/YACReader/YACReader.pro b/YACReader/YACReader.pro
+index 120ad83..ff8263f 100644
+--- a/YACReader/YACReader.pro
++++ b/YACReader/YACReader.pro
+@@ -219,7 +219,11 @@ DEFINES += "LIBDIR=\\\"$$LIBDIR\\\""  "DATADIR=\\\"$$DATADIR\\\""
+ 
+ #MAKE INSTALL
+ 
+-INSTALLS += bin docs icon desktop translation manpage
++!haiku {
++  INSTALLS += bin docs icon desktop translation manpage
++} else {
++  INSTALLS += bin docs translation manpage
++}
+ 
+ bin.path = $$BINDIR
+ isEmpty(DESTDIR) {
+@@ -228,7 +232,11 @@ isEmpty(DESTDIR) {
+   bin.files = $$DESTDIR/YACReader
+ }
+ 
+-docs.path = $$DATADIR/doc/yacreader
++isEmpty(DOCDIR) {
++  docs.path = $$DATADIR/doc/yacreader
++} else {
++  docs.path = $$DOCDIR/
++}
+ 
+ #rename docs for better packageability
+ docs.extra = cp ../README.txt ../README
+@@ -240,10 +248,18 @@ icon.files = ../YACReader.svg
+ desktop.path = $$DATADIR/applications
+ desktop.files = ../YACReader.desktop
+ 
+-translation.path = $$DATADIR/yacreader/languages
++!haiku {
++  translation.path = $$DATADIR/yacreader/languages
++} else {
++  translation.path = $$DATADIR/data/languages

```

Comment:
Wouldn't it be better to drop all the patch chunks that change `YACReader/YACReader.pro`, **except** the one with:
```diff
-INSTALLS += bin docs icon desktop translation manpage
+!haiku {
+  INSTALLS += bin docs icon desktop translation manpage
+} else {
+  INSTALLS += bin docs translation manpage
+}
```
We would even be able to pass `DATADIR="$dataDir"` instead of `DATADIR="$dataDir"/YACReader` and then get `data/yacreader/` instead of `data/YACReader/data/`.

Hmm, you might even wish to also drop the chunck which removes `icon desktop` from `INSTALLS` and, instead, remove them after the `make install`.


BTW, we also need this if we choose to install the runtimes in `apps/` instead of `apps/YACReader/`:
```diff
 defineDebugInfoPackage yacreader$secondaryArchSuffix \
-	"$appsDir"/YACReader/YACReader \
-	"$appsDir"/YACReader/YACReaderLibrary
+	"$appsDir"/YACReader \
+	"$appsDir"/YACReaderLibrary
```

Reply:
```diff
 BUILD()
 {
 	qmake \
-		PREFIX="$appsDir/YACReader" \
+		PREFIX="$prefix" \
 		MANDIR="$manDir" \
 		DOCDIR="$docDir" \
-		BINDIR="$appsDir"/YACReader \
-		LIBDIR="$appsDir"/YACReader/lib \
-		DATADIR="$appsDir"/YACReader/data \
+		BINDIR="$appsDir" \
+		DATADIR="$dataDir" \
 		.
 	make $jobArgs
 }
```
(This is a small update. The only change compared to my previous suggestion is in the line which passes `DATADIR`.)

Reply:
Any progress on this one @extrowerk ? :)

Reply:
this doesn't make much sense to me.